uli-community/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Table of Content
+
+- [Start Development](#start-development)
+- [Authentication in Web App itself](#authentication-in-web-app-itself)
+   * [Differences between the Authentication with and without the liveview](#differences-between-the-authentication-with-and-without-the-liveview)
+   * [Adjustments to make both kinds of Authentications work](#adjustments-to-make-both-kinds-of-authentications-work)
+      + [Changing the Scope of non-liveview routes](#changing-the-scope-of-non-liveview-routes)
+- [Authentication with Token](#authentication-with-token)
+- [Authorization](#authorization)
+- [Access Token](#access-token)
+
+## Start Development
+
+To start your Phoenix server:
+
+  * Run `docker compose up`
+  * Run `mix setup` to install and setup dependencies
+  * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
+
+Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+
+Ready to run in production? Please [check the deployment guides](https://hexdocs.pm/phoenix/deployment.html).
+
+## Authentication in Web App itself
+
+To generate template code for authentication, phoenix provides a command `mix phx.gen.auth`. This command provides an option to generate code with either live-view or without live-view.
+
+In this project, we wanted to keep both kinds of authentication options as a proof of concept. So we first generated the template code with the normal (with no live-view) option and then generated it with the live-view option.
+
+### Differences between the Authentication with and without the liveview
+
+- The code for non-liveview generated all the relevant routes and their respective controllers.
+- The code for liveview generated routes for the liveview templates with their respective controllers inside `/live` folder.
+- The generator did not overwrite the routes, it just added the new routes below the old ones with different macros (live instead of REST macros). However, the route addresses were the same for both the generated routes. Ex- both had `/users/log_in` addresses for their login routes.
+- For login, they both use the `UserSessionController` controller; because of this, the generator for liveview overwrote the code inside this controller. 
+- Using the generator generated the same migration file to create the Accounts table twice. (In case of overwriting a file, the generator always asks for every file).
+
+### Adjustments to make both kinds of Authentications work
+
+- The following points are only for authentication routes.
+
+#### Changing the Scope of non-liveview routes
+- As the route addresses were the same in both cases, the `scope` for the non-liveview routes was changed from `"/"` to `"/nolive"`. 
+- By default, on clicking the registration and login buttons on the homepage, everything will render the liveview templates. To access the similar route for the no-liveview templates prepend `/nolive` to the route. 
```

Comment:
A non-critical feedback. Since the server URLs are public, something like `nolive` seems odd. We'll see if we can replace this with something more proper later.